cmd/imagedigestexporter/main.go

```diff
@@ -31,6 +34,8 @@ import (
 var (
 	images                 = flag.String("images", "", "List of images resources built by task in json format")
 	terminationMessagePath = flag.String("terminationMessagePath", "/tekton/termination", "Location of file containing termination message")
+	enableSpire            = flag.Bool("enable_spire", false, "If specified by configmap, this enables spire signing and verification")
```

Comment:
do we still use `enable-spire` feature flag? thought we changed it to `enforce-nonfalsifiability` in https://github.com/tektoncd/pipeline/pull/5902 in line with the TEP?

Reply:
The feature flag which is used in the config map is enforce-nonfalsifiability.
But this is a command line flag passed into imagedigestexporter by the controller. 
This command line flag is internal and not an API for the user, as such for these internal flags, enable_spire was retained as it keeps things simpler.